conf.py

```diff
@@ -17,12 +17,68 @@
 
 import fnmatch
 from io import open
-from itertools import starmap
+from functools import lru_cache
```

Comment:
It looks like the only two things this is used for are zero-argument functions, so it might as well be replaced with a simple memoize/cache decorator.